lib/ansible/playbook/task.py

```diff
@@ -79,7 +79,7 @@ class Task(Base, Conditional, Taggable, CollectionSearch):
     _loop = FieldAttribute()
     _loop_control = FieldAttribute(isa='class', class_type=LoopControl, inherit=False)
     _notify = FieldAttribute(isa='list')
-    _poll = FieldAttribute(isa='int', default=10)
+    _poll = FieldAttribute(isa='int', default=C.DEFAULT_POLL_INTERVAL)
```

Comment:
Will anyone mind/care/notice that this has changed from 10 to 15 seconds?

Reply:
i doubt it, but its in clog now

Reply:
I think it's fine, just wanted to ask the question.